templates/cluster-template-machinepool.yaml

```diff
@@ -110,6 +106,11 @@ spec:
     mounts:
     - - LABEL=etcd_disk
       - /var/lib/etcddisk
+  machineTemplate:
```

Comment:
Is this change part of the v1alpha4 or the related to machinepool change?

Reply:
this is https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1389/commits/8f8eff9ea5144639e71215176856b69e28321a36 which is part of the capi updates (https://github.com/kubernetes-sigs/cluster-api/pull/4644)